docs/getting-started-guides/ubuntu/operational-considerations.md

```diff
@@ -116,45 +116,52 @@ juju switch default
 
 ### Running privileged containers
 
-By default juju-deployed clusters do not support running privileged containers. If you need them, you have to edit ```/etc/default/kube-apiserver``` on the master nodes, and ```/etc/default/kubelet``` on your worker nodes. 
+By default juju-deployed clusters do not support running privileged containers.
+If you need them, you have to enable the ```allow-privileged``` config on both
+kubernetes-master and kubernetes-worker:
 
-On Kubernetes Core or on small deployment, run the following commands from the Juju client: 
+```
+juju config kubernetes-master allow-privileged=true
+juju config kubernetes-worker allow-privileged=true
+```
 
-#### Manually 
+### Private registry
 
-1. Update the Master
+With the "registry" action that is part for the kubernetes-worker charm, you
```

Comment:
I think "that is part of the kubernetes-worker charm" is unnecessary, and here are a few suggestions to smooth out the wording:
"With the _registry_ action, you can easily create a private docker registry that uses TLS authentication. However, note that a registry deployed with that action is not HA; it uses storage tied to the kubernetes node where the pod is running. Consequently, if registry pod changes are migrated from one node to another, you will need to re-publish the images."